es-ds-components/components/es-carousel.vue

```diff
@@ -171,6 +259,18 @@ onMounted(() => {
         :key="key"
         :autoplay-interval="autoplayInterval"
         :circular="isMounted && props.circular"
+        class="es-carousel"
+        :class="{
+            'es-carousel--brand': props.variant === 'brand',
```

Comment:
There isn't consistency in our code on this, but I defer to not including `props.` since it's not necessary and leads to cleaner code